spec.html

```diff
@@ -29024,12 +29024,17 @@ <h1>
         <h1>
           HostEnsureCanCompileStrings (
             _calleeRealm_: a Realm Record,
+            _paramsStrings_: a List of Strings,
+            _bodyString_: a String,
+            _direct_: a Boolean,
           ): either a normal completion containing ~unused~ or a throw completion
         </h1>
         <dl class="header">
           <dt>description</dt>
           <dd>It allows host environments to block certain ECMAScript functions which allow developers to interpret and evaluate strings as ECMAScript code.</dd>
         </dl>
+        <p>_paramsStrings_ represents the strings that, when using the `Function` constructor, will be concatenated together to build the parameters list. _bodyString_ represents the function body or the string passed to an `eval` call.</p>
```

Comment:
```suggestion
        <p>_paramsStrings_ represents the strings that, when using one of the `Function`, `GeneratorFunction`, `AsyncFunction`, or `AsyncGeneratorFunction` constructors, will be concatenated together to build the parameters list. _bodyString_ represents the function body or the string passed to an `eval` call.</p>
```

or some other wording - it's not just `Function`, is the point.

Reply:
```suggestion
        <p>_paramsStrings_ represents the strings that, when using one of the function constructors, will be concatenated together to build the parameters list. _bodyString_ represents the function body or the string passed to an `eval` call.</p>
```

if we want to have it a more readable (even though less explicit) sentence